packages/zoe/CONTRIBUTING.md

```diff
@@ -46,4 +46,56 @@ Before submitting a pull request, please:
+3. `cd packages/cosmic-swingset`
+4. `rm -rf t3`
+4. `make scenario3-setup`
+5. `make scenario3-run-client`
+6. Open `http://127.0.0.1:8000/`
+8. `moolaAssayP = E(home.registrar).get("moola_2059");`
+9. `simoleanAssayP = E(home.registrar).get("simolean_9794");`
+10. `automaticRefundHandleP = E(home.zoe).install("function getExport() { 'use strict'; let exports = {}; const module = { exports }; 'use strict';\n\nObject.defineProperty(exports, '__esModule', { value: true });\n\nfunction _interopDefault (ex) { return (ex && (typeof ex === 'object') && 'default' in ex) ? ex['default'] : ex; }\n\nvar harden = _interopDefault(require('@agoric/harden'));\n\n/**\n * This is a very trivial contract to explain and test Zoe.\n * AutomaticRefund just gives you back what you put in. It has one\n * method: `makeOffer`. AutomaticRefund then tells Zoe to complete the\n * offer, which gives the user their payout through Zoe. Other\n * contracts will use these same steps, but they will have more\n * sophisticated logic and interfaces.\n * @param {contractFacet} zoe - the contract facet of zoe\n */\nconst makeContract = harden((zoe, terms) => {\n  let offersCount = 0;\n  const makeSeatInvite = () => {\n    const seat = harden({\n      makeOffer: () => {\n        offersCount += 1;\n        // eslint-disable-next-line no-use-before-define\n        zoe.complete(harden([inviteHandle]));\n        return `The offer was accepted`;\n      },\n    });\n    const { invite, inviteHandle } = zoe.makeInvite(seat, {\n      seatDesc: 'getRefund',\n    });\n    return invite;\n  };\n  return harden({\n    invite: makeSeatInvite(),\n    publicAPI: {\n      getOffersCount: () => offersCount,\n      makeInvite: makeSeatInvite,\n    },\n    terms,\n  });\n});\n\nexports.makeContract = makeContract;\n\n\nreturn module.exports;\n}\n", "getExport");`
+11. `inviteP = automaticRefundHandleP.then( automaticRefundHandle =>
+    E(home.zoe).makeInstance(automaticRefundHandle, harden({ assays:
+    [moolaAssayP, simoleanAssayP]})));`
+12. `inviteAssayP = E(home.zoe).getInviteAssay();`
+13. `exclInviteP = E(inviteAssayP).claimAll(inviteP);`
+14. `E(exclInviteP).getBalance();`
+15. `E(moolaAssayP).makeUnits(3).then(units => moola3 = units);`
+16. `E(simoleanAssayP).makeUnits(2).then(units => simoleans2 = units);`
+17. `offerRules = harden({ payoutRules: [{ kind: 'wantAtLeast', units:
+    moola3}, { kind: 'wantAtLeast', units: simoleans2 }], exitRule: {
+    kind: 'waived' }});`
+18. `seatAndPayoutP = E(home.zoe).redeem(exclInviteP, offerRules,
+    harden([undefined, undefined]));`
+19. `seatAndPayoutP.then(obj => {seat = obj.seat; payoutP = obj.payout});`
+20. `E(seatAndPayout.seat).makeOffer();`
+21. `seatAndPayout.payout.then(p => moolaPayment = p[0]);`
+20. `E(moolaPayment).getBalance();` (should be an extent of 0)
+
+
+To test that the update works with agoric-cli, do:
+
+1. Run `yarn install` from the workspace root
+2. Run `yarn build` from the workspace root
+3. `cd packages/agoric-cli`
```

Comment:
The rest of the steps (4 and onward) should be replaced with:
4. Follow instructions in README.md

They'll be updated when #443 is landed.